src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -966,7 +966,10 @@ protected override void OnPointerPressed(PointerRoutedEventArgs args)
 
 			if (ShouldFocusOnPointerPressed(args)
 				// UWP Captures pointer is not Touch
-				&& CapturePointer(args.Pointer))
+#if !__WASM__
```

Comment:
For me it sounds like if we where just hiding a bug here. The comment mention that the `TextBox` is capturing the pointer here, changing this behavior for all browsers is probably problematic.

I assume the issue is that as we capture the pointer, FF is no longer able to select the text. If so, I think that we should do "managed only capture" instead. (It's not supported yet, but it's only about not calling the [`CaptureNative`](https://github.com/unoplatform/uno/blob/master/src/Uno.UI/UI/Xaml/Internal/PointerCapture.wasm.cs#L14) in that particular case, and probably only on FF)

Also I think that this code has not been changed since a while and I suspect that the selection issue is a regression. It would be nice to determine what change introduced that regression to better understand how we could fix it.

Reply:
@dr1rrb thanks for your feedback.

Obviously it would be pretty hard - and most likely a waste of effort  - to ask an external contributor like @TopProgrammer77 to dig through the history of UNO changes in order to understand what potentially might have gone wrong in the past. 

However, it could make sense - forward thinking  - to fix the issue based one the `CaptureNative `approach you suggested. I tasked @TopProgrammer77 accordingly ...